css.js

```diff
@@ -127,26 +127,27 @@ define(function() {
   // <link> load method
   var linkLoad = function(url, callback) {
     var link = document.createElement('link');
+    link.href = url;
     link.type = 'text/css';
     link.rel = 'stylesheet';
-    if (useOnload)
-      link.onload = function() {
-        link.onload = function() {};
-        // for style dimensions queries, a short delay can still be necessary
-        setTimeout(callback, 7);
-      }
-    else
-      var loadInterval = setInterval(function() {
+    head.appendChild(link);
+
+    var loadInterval = setInterval(function () {
+      if (useOnload) {
+        link.onload = function () {
+          link.onload = function () { };
```

Comment:
```
link.onload = function () {
  link.onload = function () { };
```

This kinda scary, not ok, need to test the code properly, if it's really needed.